src/lava/lib/optimization/solvers/generic/nebm/process.py

```diff
@@ -127,8 +123,7 @@ def __init__(
 
         self.refract_counter = Var(
             shape=shape,
-            init=(refract or 0)
-            + np.right_shift(
+            init=0 + np.right_shift(
                 np.random.randint(0, 2**8, size=shape), (refract_scaling or 0)
```

Comment:
Should this random initialization be factored out of the guts of this model, or at least accept a seed parameter? As it is, I assume this makes it impossible to replicate a run with an identical seed.

Reply:
Good point. I now changed it so that the numpy-random-seed can be set as hyperparameter (defaults to 0).